config/rbac/lighthouse-agent/ocp_cluster_role_binding.yaml

```
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ocp-submariner-lighthouse-agent
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: ocp-submariner-lighthouse-agent
+subjects:
+  - kind: ServiceAccount
+    name: submariner-lighthouse-agent
+    namespace: submariner-operator
```

We shouldn't hardcode `submariner-operator`. Other similar files specify `placeholder` which the generator looks for.

Ditto for the other files.

```suggestion
    namespace: placeholder
```

---

Ooops, missed it. thanks.